Vagrantfile

```diff
@@ -11,8 +11,9 @@ Vagrant.configure("2") do |config|
   config.vm.hostname = vconfig['vagrant_hostname']
   config.vm.network :private_network, ip: vconfig['vagrant_ip']
   config.ssh.insert_key = false
+  config.ssh.forward_agent = true
```

Comment:
I would like to make this configurable as well; but still default to `false` (so if defined, set it to defined value, otherwise false).